port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -415,7 +415,7 @@ async def sync_raw_all(
             app_config = await self.port_app_config_handler.get_port_app_config(
                 use_cache=False
             )
-
+            logger.info(f"Resync will use the following mappings: {app_config.dict()}")
```

Comment:
Isn't this too much? Maybe add this as debug?

Reply:
I feel like its required, it isn't too spammy , and it will help understanding on what mapping triggered the resync